### DISCUSSION TITLE
Display genotype phasing

### DISCUSSION THREAD

**Discussion by jules-o** [broadinstitute/seqr#2034] [Q&A]
With phased calls could the genotype possibly be displayed as it appears in the VCF - e.g. 0|1 rather than 0/1? Plus the phase set where available (PID field in GATK or PS in Long Ranger). That'd be a big help in identifying compound heterozygous variants where full trio data are not available, or with de novo mutations.

**Comment by hanars** [comment]
GATK does not really support phased calls so we will not be supporting this in seqr

**Comment by karynne7** [comment]
Does Seqr support GATK-phased VCFs? I'm getting errors when I try to load one, but it's ok when I remove those lines. Just want a sanity check, thanks!
Example: 1	1254443	rs1886772	G	A	12553.9	PASS	AC=12;AF=1;AN=12;DB;DP=312;ExcessHet=3.0103;FS=0;MLEAC=12;MLEAF=1;QD=29;SOR=0.7	GT:AD:DP:GQ:PGT:PID:PL:PS	1|1:0,52:52:
99:1|1:1254436_A_G:2230,156,0:1254436	1|1:0,41:41:99:1|1:1254436_A_G:1783,123,0:1254436	1|1:0,55:55:99:1|1:1254436_A_G:2406,166,0:1254436	1|1:0,66:66:99:1|1:1254436_A_G:2788,
199,0:1254436	1|1:0,28:42:84:1|1:1254436_A_G:1223,84,0:1254436	1|1:0,49:49:99:1|1:1254436_A_G:2149,147,0:1254436

**Reply by hanars** [reply]
We do not support phased  VCFs at this time
